Resources/doc/reference/installation.rst

```diff
@@ -58,34 +65,43 @@ Next, be sure to enable the ``Page`` and ``EasyExtends`` bundles in your applica
         );
     }
 
-Before we can go on with generating our Application files trough the ``EasyExtends`` bundle, we need to add some lines which we will override later (we need them now only for the following step):
+Before we can go on with generating our Application files trough the `EasyExtendsBundle`_, we need to add some lines which we will override later (we need them now only for the following step):
+
+.. configuration-block::
 
-.. code-block:: yaml
+    .. code-block:: yaml
+
+        # app/config/config.yml
 
-    sonata_page:
-        slugify_service:   sonata.core.slugify.cocur # old BC value is sonata.core.slugify.native
-        multisite:        host
-        default_template: default # template key from templates section, used as default for
-        templates:
-            default:  { path: 'SonataPageBundle::layout.html.twig',          name: 'default' }
-            2columns: { path: 'SonataPageBundle::2columns_layout.html.twig', name: '2 columns layout' }
+        sonata_page:
+            slugify_service:   sonata.core.slugify.cocur # old BC value is sonata.core.slugify.native
+            multisite:        host
+            default_template: default # template key from templates section, used as default for
```

Comment:
the comment ends like this: `used as default for`  it looks like something is missing at the end.